sourcemash/frontend/templates/help.tpl

```diff
@@ -0,0 +1,91 @@
+<div class="help-fixed-action-btn">
+  <span class="btn-floating btn-large">?</span>
+</div>
+
+<div id="help-modal" class="modal">
+  <div class="modal-content section no-pad-bot">
+    <div class="container">
+      <div class="row">
+
+        <h2 class="col s12 light center header flow-text">Confused?</h2>
+        <h4 class="col s12 light center header flow-text">Let's get you back on track!</h4>
+        <p class="col s12 divider"><p>
+
+        <h5 class="col s12 light center header flow-text">Sourcemash categorizes your articles to help you get through your feeds faster.</h5>
+
+      </div>
+
+      <div class="row">
+        <div class="col s4">
+          <div class="center promo">
+            <a href="/#browse">
+              <i class="mdi-editor-insert-comment promo-icon"></i>
+            </a>
+          </div>
+        </div>
+        <div class="col s8">
+          <p class="promo-caption">Step 1: Source selection</p>
```

Comment:
Don't think that's necessary. I think they should know more from just reading the headers.